packages/material-design-lite/material-design-lite.scss

```diff
@@ -15,6 +15,7 @@
  */
 
 @import "mdl-animation/mdl-animation";
+@import "mdl-button/index";
```

Comment:
would you mind changing this to `mdl-button/mdl-button` to adhere to the codebase's convention?